javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/javaparsermodel/declarations/JavaParserClassDeclarationTest.java

```diff
@@ -655,9 +655,10 @@ void testGetAllMethods() {
                 .sorted(Comparator.comparing(MethodUsage::getQualifiedSignature))
                 .collect(Collectors.toList());
 
-        List<String> signatures = sortedMethods.stream().map(m -> m.getQualifiedSignature()).collect(Collectors.toList());
+        /*MED changed*/
```

Comment:
What is this comment?

Reply:
the reason I did the /** MED */ comments is because AFTER I change the packages (since the filenames change) it looks like "everything" is changed (I physically delete the old files abnd create new ones)

When I do that I wanted to keep track of the "actual" changes, so I could do a global search and verify where the manual changes took place.

Hmm. the above I believe should have been changed to Collectors.toSet() because after I change the package name and I receive the signatures the return type goes from 
"com.github.javaparser.***"
to
"org.javaparser..."

so assuming we had a method that returned a String and a CompilationUnit... the List would contain:
1) com.github.javaparser.ast.CompilationUnit (rest of signature)
2) java.lang.String (rest of signature) 
in the list...

but AFTER the name change the list order would be
1) org.javaparser.ast.CompilationUnit (rest of signature)
2) java.lang.String (rest of signature)

thats why we convert it to a Set, since both will be naturally ordered and we dont get a comparison failure (yeah I couldnt figure out why the test failed to begin with earlier until i dug in a little)
java.***



because if we test the list version (ordered) the test fails... but when we compare the a Set to another Set they are the same... (since when